sp_at/pages/integration_page.py

```diff
@@ -0,0 +1,73 @@
+from abc import ABC, abstractmethod
+
+import time
+from selenium.webdriver.remote.webdriver import WebDriver
+
+from sp_at.elements.element import ElementByXpath, ElementById
+from sp_at.elements.url import SimpleUrl, AddressOf
+from sp_at.pages import main_url
+from sp_at.pages.structures import Body, Header, MainHeader
+
+
+class _Body(Body):
+    def __init__(self, driver: WebDriver):
+        self._title = ElementByXpath(driver, '//*[@id="content_left"]/h1')
+        self._what_next_block = ElementById(driver, 'content_right')
+        self._schedule_block = ElementById(driver, 'content_left')
+        self._schedule_button = ElementByXpath(driver, '//*[@id="content_right"]/a/button')
+
+    def present(self)-> bool:
+        for element in (self._title, self._what_next_block, self._schedule_block, self._schedule_button):
+            if element.instance().is_displayed():
+                return True
```

Comment:
This line says that body is present if the title is displayed. I'm not sure that this is correct. I suppose you need to return `true` only if all elements are displayed.